### DISCUSSION TITLE
How to import a file from a react lib into a node lib?

### DISCUSSION THREAD

**Discussion by giopetris** [nrwl/nx#18967] [General]
I'm trying to use https://react.email/ in a NX monorepo. I currently have a node lib to send emails, and I'm trying to plug react email into it. 

I already tried to put the `.tsx` files into that node lib, and also extracting into a separated react lib, but none worked so far. 

I'm getting the following error after last attempt:

<img width="1375" alt="CleanShot 2023-09-01 at 11 08 14@2x" src="https://github.com/nrwl/nx/assets/1332141/45b0cf27-5a9a-40e5-9bda-fd56e7b11b14">

How can I setup it so I can send emails with https://react.email/ in a NX monorepo?

**Comment by svantepihl** [comment]
@giopetris did you ever figure it out? :)

**Comment by DeividZavala** [comment]
any updates on this?
